components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -283,23 +292,28 @@ - (void)setAccessoryView:(UIView *)accessoryView {
   [self setNeedsLayout];
 }
 
-- (void)setButtonFont:(UIFont *)font {
-  _buttonFont = font;
-
-  [self updateButtonFont];
-}
-
 - (void)updateButtonFont {
-  UIFont *finalButtonFont = self.buttonFont ?: [[self class] buttonFontDefault];
-  if (self.mdc_adjustsFontForContentSizeCategory) {
-    if (self.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable) {
-      finalButtonFont = [finalButtonFont
-          mdc_fontSizedForMaterialTextStyle:kTitleTextStyle
-                       scaledForDynamicType:self.mdc_adjustsFontForContentSizeCategory];
-    }
-  }
   for (MDCButton *button in self.actionManager.buttonsInActionOrder) {
-    [button setTitleFont:finalButtonFont forState:UIControlStateNormal];
+    if (self.mdc_adjustsFontForContentSizeCategory) {
```

Comment:
Can you move `if (self.mdc_adjustsFontForContentSizeCategory) {` outside of the loop as a guard statement? E.g.:

```objc
  if (!self.mdc_adjustsFontForContentSizeCategory) {
    return;
  }
```